packages/main/src/plugin/container-registry.ts

```diff
@@ -128,6 +131,12 @@ export class ContainerProviderRegistry {
   protected streamsPerContainerId: Map<string, NodeJS.ReadWriteStream> = new Map();
   protected streamsOutputPerContainerId: Map<string, Buffer[]> = new Map();
 
+  isUseLibpodApiEnabled(): boolean {
```

Comment:
I think the method should be for images also I find it misleading

I explain why
I may think using the method that it is to check if lipodAPI is there or not (like if using a podman connection)

But reality is that we want to use that flag for images purpose

maybe `useLibpodApiForImageList() :boolean {` or `isUsingLibpodForImages`